BackEnd/Restaurant/Api/Data/DTOs/UserDTOs/ReadUserDto.cs

```diff
@@ -10,7 +10,7 @@ public class ReadUserDto
 
         public string Phone { get; set; } = string.Empty;
 
-        public string? ImageUrl { get; set; } = string.Empty;
+        public string? Image { get; set; } = string.Empty;
```

Comment:
Why rename this to Image insted of ImageUrl ?
ImageUrl is more appropriate if you ask me

Reply:
I thought that we should make Dto's more user friendly instead of data representative, when naming